modular_skyrat/modules/morewizardstuffs/spells/magical_stimpack.dm

```diff
@@ -0,0 +1,18 @@
+/datum/action/cooldown/spell/stimpack
+	name = "Magic Stimpack"
+	desc = "This spell magically injects stimulants straight into your blood. Won't work on species with no reagent reactions!"
+	school = "transmutation"
+	cooldown_time = 10 SECONDS
+	cooldown_reduction_per_rank = 1.25 SECONDS
+	spell_requirements = NONE
+	invocation = "STIMULUS CHEQ'US"
+	invocation_type = INVOCATION_SHOUT
+
+/datum/action/cooldown/spell/stimpack/cast(mob/living/cast_on)
+	. = ..()
+	to_chat(cast_on, span_notice("Time appears to slow as your bodily functions rapidly speed up."))
```

Comment:
```suggestion
	balloon_alert(cast_on, "speeding up")
```

Reply:
I know what it does but for whatever reason, it give a undefined proc error

Reply:
oops you're right, updated